packages/story-editor/src/app/story/useStoryReducer/test/copySelectedElement.js

```diff
@@ -19,6 +19,31 @@
 import { setupReducer } from './_utils';
 
 describe('copySelectedElement', () => {
+  it('should do nothing if no selection', () => {
+    const { restore, copySelectedElement } = setupReducer();
+
+    // Set an initial state with a current page.
+    const initialState = restore({
+      pages: [
+        {
+          id: '111',
+          animations: [],
+          elements: [
+            { id: '123', isBackground: true, type: 'shape' },
+            { id: '456', x: 0, y: 0, type: 'shape' },
+            { id: '789', x: 0, y: 0, type: 'shape' },
+          ],
+        },
+      ],
+      current: '111',
+      selection: [],
+    });
+
+    const result = copySelectedElement();
+
+    expect(result).toStrictEqual(initialState);
+  });
+
   it("copies an element's styles and animations", () => {
```

Comment:
```suggestion
  it("copies an elements styles and animations", () => {
```

Reply:
Isn't that a possessive apostrophe? Seems correct to me, though I'm not a native speaker

Reply:
Not sure on that ...

Seeing it below as `elements` I would keep it consistent but NIT either way.  Feel free to ignore.

https://github.com/GoogleForCreators/web-stories-wp/pull/11729/files#diff-e66991742eb0e07e6ce0ec6d29ff9bd62c8d470a9d889c8e1c997ca10476697aL132